packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Updated additional dependencies for technical currency
+- BugFix: Updated engine to Node 16.7.0
```

Comment:
For both: End w/ a period and add a PR or issue link please.

Apply this to any changelog items that would be published in the Zowe Docs release notes.